docs/src/add-services/influxdb.md

```diff
@@ -75,3 +111,62 @@ To export your data from InfluxDB, follow these steps:
     ``` bash
     influx backup --host {{< variable "URL_FROM_STEP_2" >}} --token {{< variable "API_TOKEN_FROM_STEP_3" >}}
     ```
+
+## Upgrade to version 2.3 or later
+
+### From a previous 2.x version
+
+From version 2.3 onwards, the structure of relationships changes.
+
+If you're using a prior 2.x version, your app might currently rely on pulling the `bucket`, `org`, `api_token`,
+or `username` values available in the [`PLATFORM_RELATIONSHIPS` environment variable](../development/variables/use-variables.md#use-platformsh-provided-variables).
```

Comment:
Prior versions this would be called `user`, so it probably doesn't make sense to call it `username` here as that's what it would be changing to, not what they would be potentially using right now.